data/settings.js

```diff
@@ -5534,31 +5534,31 @@ If no value is specified, attachment saving to external files is disabled.
 [[variable,mail-user]] can be used.`
 	},
 
-	mail_attachment_fs: {
-		default: 'sis posix',
-		seealso: [ 'mail_attachment_dir' ],
-		values: setting_types.ENUM,
-		values_enum: [ 'posix', 'sis posix', 'sis-queue posix' ],
+	mail_attachment: {
+		values: setting_types.NAMED_FILTER,
+		seealso: [ 'fs_driver', 'mail_attachment_dir' ],
 		text: `
-Which filesystem type to use for saving attachments.
+Named filter for initializing [[link,fs,FS Driver]] for external attachments.
 
-Options:
+Commonly used options:
 
 \`posix\`
 :   No single-instance storage (SiS) done (this option might simplify the
```

Comment:
This really should be SIS everywhere, not SiS..

Reply:
OK. Need to find a way to prevent others from changing this back again (it's been changed once from `SIS` -> `SiS` before).